kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/wires/WiresPanelUi.ui.xml

```diff
@@ -11,36 +9,173 @@
 
     Contributors:
       Eurotech
-
+      Amit Kumar Mondal (admin@amitinside.com)
```

Comment:
As discussed at EclipseCon, just use your name. If someone at Eurotech did co-author on that file add them in the contributors list and add "and others".